dr-multi-replica.md

```diff
@@ -0,0 +1,208 @@
+
+1. 创建类似于以下的集群拓扑文件：
+
+    ```toml
+    global:
+      user: "root"
+      ssh_port: 22
+      deploy_dir: "/data/tidb_cluster/tidb-deploy"
+      data_dir: "/data/tidb_cluster/tidb-data"
+
+    server_configs:
+      tikv:
+        server.grpc-compression-type: gzip
+      pd:
+        replication.location-labels:  ["Region","AZ"] # PD 会根据 TiKV 节点的 Region 和 AZ 配置来进行副本的调度。
+
+    pd_servers:
+      - host: tidb-dr-test1
+        name: "pd-1"
+      - host: tidb-dr-test2
+        name: "pd-2"
+      - host: tidb-dr-test3
+        name: "pd-3"
+      - host: tidb-dr-test4
+        name: "pd-4"
+      - host: tidb-dr-test5
+        name: "pd-5"
+
+    tidb_servers:
+      - host: tidb-dr-test1
+      - host: tidb-dr-test3
+
+
+    tikv_servers:  # 在 TiKV 节点中通过 labels 选项来对每个 TiKV 节点所在的 Region 和 AZ 进行标记
+      - host: tidb-dr-test1
+        config:
+          server.labels: { Region: "Region1", AZ: "AZ1" }
+      - host: tidb-dr-test2
+        config:
+          server.labels: { Region: "Region1", AZ: "AZ2" }
+      - host: tidb-dr-test3
+        config:
+          server.labels: { Region: "Region2", AZ: "AZ3" }
+      - host: tidb-dr-test4
+        config:
+          server.labels: { Region: "Region2", AZ: "AZ4" }
+      - host: tidb-dr-test5
+        config:
+          server.labels: { Region: "Region3", AZ: "AZ5" }
+
+          raftstore.raft-min-election-timeout-ticks: 1000
+          raftstore.raft-max-election-timeout-ticks: 1200
+
+    monitoring_servers:
+      - host: tidb-dr-test2
+
+    grafana_servers:
+      - host: tidb-dr-test2
+
+    alertmanager_servers:
+      - host: tidb-dr-test2
+      ```
+
+    在上面的配置中，使用了以下一系列配置来针对跨区域容灾场景进行优化：
+
+    - 使用 `server.grpc-compression-type`：gzip 启用 TiKV 之间的消息压缩，从而降低网络流量。
+    - 使用 `raftstore.raft-min-election-timeout-ticks` 和 `raftstore.raft-max-election-timeout-ticks` 延长区域 3 参加选举的时间，从而避免该区域中的副本被选举为主节点。
+
+2. 使用上面的配置文件创建集群：
+
+    ```toml
+    # tiup cluster deploy drtest v6.4.0 ./topo.yaml
+    # tiup cluster start drtest --init
+    # tiup cluster display drtest
+    ```
+
+    对集群的副本数和 Leader 限制进行配置：
+
+    ```toml
+    # tiup ctl:v6.4.0 pd config set max-replicas 5
+    # tiup ctl:v6.4.0 pd config set label-property reject-leader Region Region3
+
+    # 下面的步骤用于向集群中添加一些测试数据，可选
+    # tiup bench tpcc  prepare -H 127.0.0.1 -P 4000 -D tpcc --warehouses 1
+    ```
+
+    指定 PD leader 的优先级:
+
+    ```toml
+    # tiup ctl:v6.4.0 pd member leader_priority  pd-1 4
+    # tiup ctl:v6.4.0 pd member leader_priority  pd-2 3
+    # tiup ctl:v6.4.0 pd member leader_priority  pd-3 2
+    # tiup ctl:v6.4.0 pd member leader_priority  pd-4 1
+    # tiup ctl:v6.4.0 pd member leader_priority  pd-5 0
+    ```
+
+    > **说明：**
+    >
+    > 优先级数值越大的节点成为 leader 的可能性越高。
+
+3. 创建 placement rule，并将测试表的主副本固定在 Region 1：
+
+    ```sql
+    -- 创建两个 placement rules，第一个是 Region1 作为主 Region，在系统正常时使用，第二个是 Region2 作为备 Region。
+    -- 作为主 Region，当 Region1 出现问题时，Region2 会作为主 Region。
```

Comment:
These two lines are quite confusing. What is the second rule exactly?

Reply:
@allengaoo PTAL